es6-template-literals/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = ' Irvine Center Drive';
+    const address = `${number} ${street}`;
+
+    console.log('LearningFuze:', address);
```

Comment:
This outputs something different from the original version:

---

![image](https://user-images.githubusercontent.com/16143771/127084766-d22ef121-95da-41c7-a19f-e7739d224690.png)

---

![image](https://user-images.githubusercontent.com/16143771/127084787-34ffbe12-de12-4710-b133-25cce40c5618.png)

---

![image](https://user-images.githubusercontent.com/16143771/127084811-074d08a4-5008-4eaf-8335-d4d3589446a8.png)

Reply:
fixed

Reply:
![image](https://user-images.githubusercontent.com/75236087/127085075-79b19ceb-feec-4515-8af7-aae4be8b9b0a.png)